src/Andoromeda.Kyubey.Portal/Services/TokenTool.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Andoromeda.Kyubey.Portal.Services
+{
+    public class TokenTool
+    {
+        public static string GetTokenIncubatorBannerUri(string tokenId, string filePath)
+        {
+            if (string.IsNullOrWhiteSpace(filePath))
+            {
+                return Path.Combine("img", "null.png");
+            }
+            string fileName = Path.GetFileName(filePath);
+            return $"/Token/tokenbanner/{tokenId}/{fileName}";
```

Comment:
Why return this? Did you mapped the token_assets?

Reply:
improve it later